nwb_conversion_tools/gui/classes/forms_basic.py

```diff
@@ -50,9 +50,11 @@ def fill_fields_info(self):
             # Required fields get a red star in their label
             if 'default' not in field:
                 required = True
+                default = 'required: '+ str(field['type'])
```

Comment:
is this auto-filled?

Reply:
only for the placeholder to display the grey text not the actual field, which remains empty for the user to fill